pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -96,14 +100,17 @@ type BundleSource struct {
 
 // BundleTarget is the target resource that the Bundle will sync all source
 // data to.
+// +structType=atomic
```

Comment:
@erikgb I don't think this has to be atomic, right?

Reply:
Agree, will fix.

Reply:
Fixed